website/docs/docs/cloud/use-visual-editor.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Use the visual editor" 
```

Comment:
sugg: to describe what can be done with the editor and also less repetition since this page is underneath the "Visual editor" top-level sidebar 

```suggestion
title: "Edit and create dbt models" 
```

Reply:
not going to update the filename though as it could be confusing with the overall edit and create dbt models workflow.